src/components/EditableLabel/EditableLabel.tsx

```diff
@@ -1,23 +1,32 @@
+import { useEffect } from "react";
 import { CSSProperties, FormEvent, KeyboardEvent, useRef, useState } from "react";
```

Comment:
```suggest
import { CSSProperties, FormEvent, KeyboardEvent, useRef, useEffect, useState } from "react";
```

Reply:
whoops, thanks for the catch!

Reply:
https://github.com/ryonryon/trello-clone/pull/67/commits/82ab4f07f00f6634a7ee3423e3915cf45d9a0a97